firmware/applications/SignBuddy/tsc.c

```diff
@@ -53,30 +123,73 @@ static void hw_init(void)
   TSC_CLK_EN();
   TSC_GPIO_CLK_EN();
 
-  sampler_pin_init(TSC_SAMPLER_PIN, TSC_SAMPLER_PORT);
-  electrode_pin_init(TSC_ELECTRODE_PIN, TSC_ELECTRODE_PORT);
+  electrode_pin_init(SENSOR_PIN(3, 0), SENSOR_PORT(3, 0));
+  electrode_pin_init(SENSOR_PIN(4, 0), SENSOR_PORT(4, 0));
+  electrode_pin_init(SENSOR_PIN(4, 1), SENSOR_PORT(4, 1));
+  electrode_pin_init(SENSOR_PIN(5, 0), SENSOR_PORT(5, 0));
+  electrode_pin_init(SENSOR_PIN(5, 1), SENSOR_PORT(5, 1));
+  electrode_pin_init(SENSOR_PIN(6, 0), SENSOR_PORT(6, 0));
+  electrode_pin_init(SENSOR_PIN(6, 1), SENSOR_PORT(6, 1));
+  electrode_pin_init(SENSOR_PIN(7, 0), SENSOR_PORT(7, 0));
```

Comment:
I think this looks a little weird couldn't we just use the macros defined in the header directly like TOUCH_1_PIN, TOUCH_1_PORT it'll be easier to trace the code associated with each touch sensor

Reply:
let me know if you still think this after what I said

Reply:
Yeah I see how it simplifies things, I'm good with this